### DISCUSSION TITLE
Coding ADI

### DISCUSSION THREAD

**Discussion by abbydearellano** [now-i-know-my-abcd/docs#52] [Stats & Code Help] [answered]
Hi everyone, 

I would love guidance on coding ADI addresses for the 5.0 data release. What is the best way to reflect that participants may spend different amounts of time at their primary addresses (e.g., weekends vs. weekdays) when running analyses? Should I simply take the average of ADI scores for their addresses or is there a better way to summarize these scores? 

Thank you so much!

**Comment by kharloews** [comment, marked as answer]
Hi hi! Sorry for the delayed response on this! I think there are a couple of things that you could do. 1) Most researchers will go with the primary address and the associated ADI score, and state it as a limitation. 2) There is a variable available where informants provided the amount of time spent at each address, which you could then use to do a weighted average (based on how long they spend at each place). However, if you go through this variable, you'll see that for some participants the values don't add up to 100% or they may go over, so this can be unreliable as well. The SOP at the time wasn't able to catch that, but there is a new residential address SOP being rolled out to participants that should do a better job at catching errors like that (although those validated data won't be available until a release or two). 

There isn't a perfect solution here, but as long as you're transparent in your manuscript, it should be OK (as this is the case with all ABCD analyses using LED data). 

Hope that was helpful! 

Carlos Cardenas-Iniguez
Co-chair, LED- Environment and Policy Working Group

**Reply by abbydearellano** [reply]
Thank you so much, Dr. Cardenas-Iniguez! I appreciate your help.

**Reply by abbydearellano** [reply]
Hi Dr. Cardenas-Iniguez! 

Do you know if additional ADI data from the 1 Year Follow Up will be included in the next data release? I would love to use it for my thesis but am not sure if it will be available. Thank you!
